dask-gateway-server/Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM python:3.8.6-slim-buster as dependencies
+FROM python:3.8.11-slim-buster as dependencies
 LABEL MAINTAINER="Jim Crist-Harif"
 
 RUN apt-get update \
     && apt-get install -y tini \
-    && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
 RUN pip install --no-cache-dir \
-    aiohttp==3.7.2 \
+    aiohttp==3.7.4 \
     colorlog \
     cryptography \
-    traitlets==4.3.3 \
+    traitlets==5.0.5 \
     pyyaml \
-    kubernetes-asyncio==11.0.0
+    kubernetes-asyncio==12.1.1
```

Comment:
Systematic version updates as I had to update aiohttp in order to have a arm64 (aka. aarch64) compatible conda package in the dask-gateway image.